pkg/commands/secretstore/secret_test.go

```diff
@@ -0,0 +1,426 @@
+package secretstore_test
+
+import (
+	"bytes"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"runtime"
+	"testing"
+
+	"github.com/fastly/cli/pkg/app"
+	"github.com/fastly/cli/pkg/commands/secretstore"
+	"github.com/fastly/cli/pkg/mock"
+	"github.com/fastly/cli/pkg/testutil"
+	"github.com/fastly/go-fastly/v7/fastly"
+)
+
+func TestCreateSecretCommand(t *testing.T) {
+	const (
+		storeID      = "store123"
+		secretName   = "testsecret"
+		secretDigest = "digest"
+		secretValue  = "the secret"
+	)
+
+	tmpDir := t.TempDir()
+	secretFile := path.Join(tmpDir, "secret-file")
+	if err := os.WriteFile(secretFile, []byte(secretValue), 0x777); err != nil {
+		t.Fatal(err)
+	}
+	doesNotExistFile := path.Join(tmpDir, "DOES-NOT-EXIST")
+
+	scenarios := []struct {
```

Comment:
Not required for this PR, but if you have any experience with the new fuzzing features in go1.18 it would be great to see an example test that utilises that feature.